From 12e88aeda97251ffab436c3e5854db42ebbfc9ff Mon Sep 17 00:00:00 2001
From: Evgeni Golov <evgeni@debian.org>
Date: Fri, 25 Jul 2014 08:00:11 +0200
Subject: [PATCH] process the scripts directory on non-Linux systems

Bash and Zsh completion can be useful on non-Linux systems too.
The generated debian/{postinst/prerm} do not hurt either.
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 84cf4d1..c30fd39 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,10 +48,11 @@ if(NOT WIN32)
 
 	if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 		add_subdirectory(driver)
-		add_subdirectory(scripts)
 		add_definitions(-DHAS_CAPTURE)
 	endif()
 
+	add_subdirectory(scripts)
+
 	if(CMAKE_SYSTEM_NAME MATCHES "SunOS")
 		set(CMD_MAKE gmake)
 	else()
-- 
2.0.1

